src/components/Widgets/DateTimeControl.js

```diff
@@ -4,8 +4,12 @@ import DateTime from 'react-datetime';
 
 export default class DateTimeControl extends React.Component {
   componentDidMount() {
-    if (!this.props.value) {
-      this.props.onChange(new Date());
+    const { value, onChange } = this.props;
+    if (value == null) {
+      onChange(new Date());
+    }
+    if (value === false) {
```

Comment:
Any reason not to just use an empty string since that's the value you're going for?

Reply:
Ah, possibly not really (I to-ed and frow-ed on this a bit).

But, it's the distinction of:

`{label: "Expiry", name: "expires", widget: "date", default: false, required: false}`

vs

`{label: "Expiry", name: "expires", widget: "date", default: '', required: false}`

And to me the `false` version read more cleanly (ie, it conveyed the intent more clearly).

Happy to change it over, but personally I prefer the `false` at the moment.